### PR TITLE
NOISSUE - Add readers pagination in SDK

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: sdk/go/sdk.go [hunk 1]
```diff
@@ -168,7 +166,7 @@ type SDK interface {
 	SendMessage(chanID, msg, token string) error
 
 	// ReadMessages read messages of specified channel.
-	ReadMessages(chanID, token string) ([]mainflux.Message, error)
+	ReadMessages(chanID, token string) (messagesPageRes, error)
```
Comment (anovakovic01) [inline sdk/go/sdk.go]: You cannot use a private type for an interface return value. Please replace this with some public type and use `messagesPageRes` only internally in SDK in order to deserialize HTTP response.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**